assets/airports/cyow.json

```diff
@@ -0,0 +1,1769 @@
+                "14": [["CYRIL", "A120-"], "CAVUR", "LITRI", ["APMUP", "A50|S210-"], "MERKO", ["ILOGI", "A35+"], "PETLU", "#320"],
+                "25": [["CYRIL", "A120-"], ["TEXEN", "A50|S230-"], ["ADVOS", "A30+|S210-"], "KAREB", "#084"],
+                "32": [["CYRIL", "A80-"], ["TEXEN", "A30+"], "#320"]
+            },
+            "draw": [
+                ["ART*", "CYRIL"],
+                ["BUGSY*", "SAVAL", "TAPVO", "CYRIL"],
+                ["CYRIL", "TEXEN", "ADVOS"],
+                ["CYRIL", "CAVUR", "LITRI", "APMUP", "MERKO", "ILOGI", "PETLU"],
+                ["CAVUR", "AMAGI", "NOVUX", "TILOG", "EBNAP"]
+            ]
+        },
+        "LEAMY3": {
+            "icao": "LEAMY3",
+            "name": "Leamy Three",
+            "entryPoints": {
+                "BEMOG": [["BEMOG", "A210-"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["ALIDO", "A160-"], ["DUNUM", "A130-"], "GABEV", ["KEMLU", "A50|S210-"], "SENNZ", ["MODON", "A30+"], "VINRA", "#251"],
+                "14": [["ALIDO", "A100-"], "DUNUM", ["RAFNO", "A30+"]],
+                "25": [["ALIDO", "A160-"], ["DUNUM", "A130-"], "MERKO", ["REVAL", "A50|S210-"], ["XOLTI", "A30+"], "IGTAD", "#072"],
+                "32": [["ALIDO", "A160-"], ["DUNUM", "A130-"], "MERKO", ["ILUBO", "A50|S210-"], ["WATTO", "A30+"], "TIDUL", "#141"]
+            },
+            "draw": [
+                ["BEMOG", "ALIDO", "DUNUM", "RAFNO"],
+                ["DUNUM", "GABEV", "KEMLU", "SENNZ", "MODON", "VINRA"],
+                ["DUNUM", "MERKO", "REVAL", "XOLTI", "IGTAD"],
+                ["MERKO", "ILUBO", "WATTO", "TIDUL"]
+            ]
+        },
+        "MEECH1": {
+            "icao": "MEECH1",
+            "name": "Meech One",
+            "entryPoints": {
+                "SMARE": ["SMARE", ["TOXIP", "A210+"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["ONDOB", "A200-"], "GABEV", ["KEMLU", "A50|S210-"], "SENNZ", ["MODON", "A30+"], "VINRA", "#251"],
+                "14": [["ONDOB", "A100-"], ["RAFNO", "A30+"]],
+                "25": [["ONDOB", "A200-"], ["GABEV", "A130-"], "MERKO", ["REVAL", "A50|S210-"], ["XOLTI", "A30+"], "IGTAD", "#072"],
+                "32": [["ONDOB", "A200-"], ["GABEV", "A130-"], "MERKO", ["ILUBO", "A50|S210-"], ["WATTO", "A30+"], "TIDUL", "#141"],
+                "22": []
+            },
+            "draw": [
+                ["SMARE", "TOXIP", "ONDOB", "RAFNO"],
+                ["ONDOB", "GABEV", "KEMLU", "SENNZ", "MODON", "VINRA"],
+                ["GABEV", "MERKO", "REVAL", "XOLTI", "IGTAD"],
+                ["MERKO", "ILUBO", "WATTO", "TIDUL"]
+            ]
+        },
+        "RIVER2": {
+            "icao": "RIVER2",
+            "name": "River Two",
+            "entryPoints": {
+                "ALSET": [["YUL", "A280-"], "ALSET"],
```

Comment:
YUL is not part of the ALSET transtition.
ALSET can be moved into the body section

Reply:
So how does that work? ALSET itself is also a transition?

Reply:
The whole thing should look like this:
```javascript
"entryPoints": {
    "ALSET": [],
    "CATOG": ["CATOG", "OBTAX", ["YUL", "A280-"]],
    "VILRO": ["VILRO", ["YUL", "A280-"]]
 },
"body": ["ALSET"],
```

Reply:
Ah right cheers